docs/pages/repo/docs/core-concepts/monorepos/skipping-tasks.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Skipping tasks
+description: Never do the same work twice.
+---
+
+import Callout from '../../../../../components/Callout'
+
+# Skipping Tasks
+
+[Build caching](/repo/docs/core-concepts/caching) can dramatically speed up your build tasks - but you can do even better by using `npx turbo-ignore`. If your workspace is unaffected by your code changes, you can completely skip executing a task altogether.
+
+## Using `turbo-ignore`
+
+To skip unaffected work, ensure that your git history is available and run `npx turbo-ignore`.
+
+`turbo-ignore` uses a combination of the `--filter` and `--dry=json` flags to find changes from the parent commit to the current commit to identify affected packages. By default, `turbo-ignore` finds the difference for the **build task in the current working directory**, but you can [customize this behavior with flags](#customizing-behavior).
+
+Here's an example of the command that will be built and ran:
+
+```bash
+npx turbo run build --filter=@example/web...[3c8387ffd98b751305fe3f0284befdd00cbd4610] --dry=json
```

Comment:
Maybe this is me but this command doesn't work for me? Can someone else verify?

Reply:
Good callout, the braces were incorrect!